src/test/java/org/ays/registrationapplication/tests/AdminRegistrationApplicationCompleteTest.java

```diff
@@ -96,8 +96,8 @@ public void completeApplicationRegistrationWitInvalidLastName(String invalidFLas
                 .body("subErrors[0].type", equalTo("String"));
     }
 
-    @Test(groups = {"Regression"}, dataProvider = "invalidEmailForAdminRegistration", dataProviderClass = AysDataProvider.class)
-    public void completeApplicationRegistrationWitInvalidEmail(String invalidEmail, AysErrorMessage errorMessage) {
+    @Test(groups = {"Regression"}, dataProvider = "invalidEmail", dataProviderClass = AysDataProvider.class)
```

Comment:
Standart olması adına `invalidEmail` yerine `invalidEmailAddress` olarak isimlendirsek daha iyi olmaz mı?
```suggestion
    @Test(groups = {"Regression"}, dataProvider = "invalidEmailAddress", dataProviderClass = AysDataProvider.class)
```